base_app/src/app/cmdsets/command-button.ts

```diff
@@ -138,7 +139,7 @@ export class CommandButtonComponent implements OnChanges, OnDestroy {
     }
 
     getQuestionText(): string {
-        let qText = 'Perform unknown action?';
+        let qText: string;
```

Comment:
I think I prefer the 'Perform unknown action' initialization, since it tells us if we've missed getting properties that will set the value that's displayed.

Reply:
added this change in my most recent commit